framework/src/transfers/MultiAppConservativeTransfer.C

```diff
@@ -48,7 +48,9 @@ MultiAppConservativeTransfer::validParams()
 
 MultiAppConservativeTransfer::MultiAppConservativeTransfer(const InputParameters & parameters)
   : MultiAppFieldTransfer(parameters),
-    _from_var_names(getParam<std::vector<VariableName>>("source_variable")),
+    _from_var_names(isParamValid("source_variable")
```

Comment:
This is a required param... when is it not valid? Does it get hidden elsewhere?

Reply:
required yet suppressed for user object transfer that inherit this class
I could see a rework of parameters forcing us to re-design this

Reply:
This is okay for now, I just wanted to make sure that was the case